### ISSUE TITLE
Remove battery icon

### ISSUE THREAD

**Issue by modem7** [trip5/ESPHome-eInk-Boards#2]
Heya, 

Is there a way to globally state that you're not using a batter, but instead have the device plugged in?

Not quite sure which parts would need removing if only using USB power.

Thanks!

**Comment by trip5** [comment]
Of course.  Actually that part is pretty easy.  Once you remove the battery section from the `sensor:` section, trying to compile will throw errors and point out exactly which lines are pointing to a sensor that no longer exists.

The last few lines of the lambda are the part that draw the battery icon, so there's that too (but again, it will throw an error because it's pointing to a variable that no longer exists).

Or you could leave everything in and just comment the part that draws the battery so it doesn't show the icon.  It doesn't hurt to leave it in if you change your mind later.

You may prefer that the device no longer sleeps, too, but that's up to you... leaving it in doesn't hurt either since you can just set the Home Assistant Helper for Disable Deep Sleep as always on.
